krun/tests/test_util.py

```diff
@@ -0,0 +1,130 @@
+from krun import LOGFILE_FILENAME_TIME_FORMAT
+from krun.util import (should_skip, format_raw_exec_results, output_name,
+                       log_name, fatal, read_config, run_shell_cmd,
+                       read_results, dump_results,
+                       check_and_parse_execution_results,
+                       audits_same_platform, ExecutionFailed)
+
+import bz2
+import json
+import os
+import pytest
+import time
+
+def test_should_skip():
+    config = dict([("SKIP", ["*:PyPy:*",
+                             "*:CPython:*",
+                             "*:Hotspot:*",
+                             "*:Graal:*",
+                             "*:LuaJIT:*",
+                             "*:HHVM:*",
+                             "*:JRubyTruffle:*",
+                             "*:V8:*",
+    ])])
+    assert should_skip(config, "nbody:HHVM:default-php")
+    assert not should_skip(dict([("SKIP", [])]), "nbody:HHVM:default-php")
+
+def test_read_config():
+    path = "examples/example.krun"
+    config = read_config(path)
+    assert config is not None
+    assert config["BENCHMARKS"] == {"dummy": 1000, "nbody": 1000}
+    assert config["N_EXECUTIONS"] == 2
+    assert config["SKIP"] == []
+    assert config["MAIL_TO"] == []
+    assert config["ITERATIONS_ALL_VMS"] == 5
+    assert config["HEAP_LIMIT"] == 2097152
+
+def test_output_name():
+    assert output_name(".krun") == "_results.json.bz2"
+    assert output_name("example.krun") == "example_results.json.bz2"
+
+def test_log_name():
```

Comment:
I haven't touched log files at all. I can see that concatenating log files might make sense in reboot mode, but if you resume after a long gap I'm not so sure.